website/docs/stitch-combining-schemas.md

```diff
@@ -97,48 +94,112 @@ export interface SubschemaConfig {
+  }
+});
+```
 
-By default, schema stitching will override type definitions that are duplicated across subschemas&mdash;always favoring the final definition of fields, arguments, and descriptions for a type found in the `subschemas` array. This works fine when subschemas implement identical versions of an object type. For divergent type definitions, you may now enable [type merging](/docs/stitch-type-merging) (as of GraphQL Tools 5) to smartly merge partial type definitions from across subschemas.
+Practically speaking, binary resolution requires subschemas to implement identical versions of a type for parity.
 
 ## Adding transforms
 
-Another strategy to avoid conflicts while combining schemas is to modify one or more of the schemas using [transforms](/docs/schema-wrapping). Transforming allows a schema to be groomed in such ways as adding namespaces, renaming types, or removing fields (to name a few) prior to stitching it into the combined gateway schema. As of GraphQL Tools version 5, these transforms should be added directly to subschema config:
+Another strategy to avoid conflicts while combining schemas is to modify one or more of the schemas using [transforms](/docs/schema-wrapping#transform). Transforming allows a schema to be groomed in such ways as adding namespaces, renaming types, or removing fields (to name a few) prior to stitching it into the combined gateway schema. These transforms should be added directly to subschema config:
 
 ```js
 import { FilterRootFields, RenameTypes } from '@graphql-tools/wrap';
 
-const chirpSubschema = {
-  schema: chirpSchema,
+const postsSubschema = {
+  schema: postsSchema,
   transforms: [
-    new FilterRootFields((operation, rootField) => rootField !== 'chirpsByAuthorId'),
-    new RenameTypes((name) => `Chirp_${name}`),
+    new FilterRootFields((operation, rootField) => rootField !== 'postsByUserId'),
+    new RenameTypes((name) => `Post_${name}`),
   ],
 };
 ```
 
-In the example above, we transform the `chirpSchema` by removing the `chirpsByAuthorId` root field and adding a `Chirp_` prefix to all types. These modifications will only be present in the combined gateway schema.
+In the example above, we transform the `postsSchema` by removing the `postsByUserId` root field and adding a `Post_` prefix to all types in the schema. These modifications will only be present in the combined gateway schema.
+
+Note that when [merging types](#merged-types), all transforms are applied _prior_ to merging. That means transformed types will merge based on their transformed names within the combined gateway schema.
+
+## Error handling
+
+Whether you're [merging types](/docs/stitch-type-merging), using [schema extensions](/docs/stitch-schema-extensions), or simply combining schemas, any errors returned by a subschema will flow through the stitching process and report at their mapped output positions. It's fairly seamless to provide quality errors from a stitched schema by following some basic guidelines:
```

Comment:
Love this section

Reply:
Error handling has proven itself to be remarkably good after I spent a month going through all of our apps and backfilling common-sense errors into all of their implementations. Absolutely deserves a mention in docs... both as a vote of confidence in mature stitching capabilities, and the recognition that devs need to put in the elbow grease on their own apps.